SciELO.js

```diff
@@ -114,10 +113,19 @@ function scrape(doc, url) {
 						.split(";")
 						.map(function(x) { return x.trim(); })
 						.map(function(y) { return y.charAt(0).toUpperCase() + y.slice(1); });
-	}
-
-		item.libraryCatalog = "SciELO"
+		}
+				item.libraryCatalog = "SciELO"
+		var domAbstract = ZU.xpath(doc, '/html/body/div[1]/div[2]/div[2]/div/a[1]');
+		for (let d in domAbstract) {
+			var lookupAbstract = domAbstract[d].href;//Z.debug(lookupAbstract)	
+		}
+		ZU.processDocuments(lookupAbstract, function (scrapeAbstract){
```

Comment:
`ZU.processDocuments()`is usually used to process a multiple selection (https://www.zotero.org/support/dev/translators/coding). Since `item` is directly available in the `itemDone` the code of the callback can be executed directly.

Reply:
I have commented in code